provision.py

```diff
@@ -158,7 +158,7 @@ def setup_virtualenv(target_venv_path, requirements_file, virtualenv_args=None):
     # type: (str, str, List[str]) -> None
 
     # Check if a cached version already exists
-    output = subprocess.check_output(['sha1sum', requirements_file])
+    output = subprocess.check_output([os.path.join('tools', 'hash_reqs.py'), requirements_file])
```

Comment:
I just posted on https://github.com/isaacs/github/issues/386 about this issue, since it's been getting pretty annoying.